core/src/main/java/org/mule/construct/AbstractPipeline.java

```diff
@@ -91,6 +92,34 @@ protected MessageProcessor createPipeline() throws MuleException
         return builder.build();
     }
 
+    /**
+     * A fallback method for creating a {@link ProcessingStrategy} to
+     * be used in case the user hasn't specified one through either
+     * {@link #setProcessingStrategy(ProcessingStrategy)},
+     * through {@link MuleConfiguration#getDefaultProcessingStrategy()}
+     * or the {@link MuleProperties#MULE_DEFAULT_PROCESSING_STRATEGY}
+     * system property
+     *
+     * @return a {@link SynchronousProcessingStrategy}
+     */
+    protected ProcessingStrategy createDefaultProcessingStrategy()
+    {
+        return new SynchronousProcessingStrategy();
```

Comment:
this method only gets invoked if the mule configuration doesn't specify a default. Look at the initialiseProcessingStrategy() method